SwiftSampleProject/SwiftyVokoder/VOKManagedObjectMap+Swift.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  VOKManagedObjectMap+Swift.swift
+//  SwiftSampleProject
+//
+//  Created by Carl Hill-Popper on 11/13/15.
+//  Copyright © 2015 Vokal.
+//
+
+import Vokoder
+
+extension VOKManagedObjectMap {
+
+    /**
+     Creates a managed object map from a foreign key and String-based enum case
+     */
+    convenience init<StringEnum: RawRepresentable where StringEnum.RawValue == String>
```

Comment:
Yes - `RawValue` in the where clause is checking the class of the StringEnum and below it's getting the raw value of the actual key itself. Also OO STEALING THIS.